### ISSUE TITLE
expo-linear-gradient incorrect color normalization on web

### ISSUE THREAD

**Issue by marnusw** [expo/expo#5724]
### Environment

```
Expo CLI 3.1.0 environment info:
    System:
      OS: Windows 10
    Binaries:
      Yarn: 1.15.2 - C:\Program Files (x86)\Yarn\bin\yarn.CMD
      npm: 6.4.1 - C:\Program Files\nodejs\npm.CMD
```

This is a regression in `v7.0.0`, everything works fine when reverting to `v6.0.0`.

### Steps to Reproduce

Look at `LinearGradient-test.web.tsx` and the snapshop test results in `LinearGradient-test.web.tsx.snap.web`.

### Expected Behavior

Named and Hex colors should map to proper `rgba` colors for the gradient.

### Actual Behavior

Compare the test and the generated snapshot:

```js
it(`renders a multi-color gradient background with alpha`,
colors={['cyan', '#ff00ff', 'rgba(0,0,0,0)']

// generates

"linear-gradient(180deg, rgba(255,0,255,1.00),rgba(255,255,0,1.00),rgba(0,0,0,0.00))"
```

The first two colors in the result are incorrect, the correct values would be:

```js
'cyan' => rgb(0,255,255,1.00)
'#ff00ff' => rgba(255,0,255,1.00)
```

### Reproducible Demo

N/A, the test in the project itself is invalid.

**Comment by Florianer** [comment]
Second that. Following the documentation (https://docs.expo.io/versions/latest/sdk/linear-gradient/) will generate a red-pinkish Facebook button. The linear gradient in 6.0.0 is from blue to white top-down and still working in 7.0.0 on the tablet, whereas in web it's looking red-pinkish fading to white.

**Comment by Florianer** [comment]
Not sure why and I also have no knowledge on colors, but if I comment out [processColor](https://github.com/necolas/react-native-web/blob/0.8.0/packages/react-native-web/src/exports/processColor/index.js) line 24
`  // int32Color = ((int32Color << 24) | (int32Color >>> 8)) >>> 0;`
then at least what I can look at, the colors are matching again. But the file was not changes after [0.8.0](https://github.com/necolas/react-native-web/commits/0.11.7/packages/react-native-web/src/exports/processColor/index.js), the copyright header does not count. But at this time it was necolas [fix] processColor compatibility with React Native ff5a928 on 29 May 2018. Maybe a breaking change happened from 6.0.0 to 7.0.0 in expo-linear-gradient or in RN? I'm not experienced to dig further around, but maybe as a starting point for the contributors. Until a proper fix, I'll keep this line commented out, as long as I do not see other side effects.

**Comment by EvanBacon** [comment]
https://github.com/expo/expo/pull/5843

**Comment by EvanBacon** [comment]
fix is live in `expo-linear-gradient@7.0.1` really sorry for the inconvenience
